apps/alf/alf.cpp

```diff
@@ -74,7 +74,7 @@ int main(int argc, const char * argv[])
     // Options Section: Input / Output parameters.
     addSection(parser, "Input / Output");
     addOption(parser, seqan::ArgParseOption("i", "input-file", "Name of the multi-FASTA input file.", seqan::ArgParseArgument::INPUT_FILE));
-    setValidValues(parser, "input-file", "fa fasta");
+    setValidValues(parser, "input-file", seqan::SeqFileIn::getFileExtensions());
```

Comment:
Does this work? Usually, we need to add a string literal comma separated to the `setValidvalues`? I mean, I am happy if not 😁

Reply:
Yes it works because the function `setValidValues()` also takes a vector. See [here](https://github.com/seqan/seqan/blob/c16542da02eb28dcfe01c255138dd3f19a0d72f7/include/seqan/arg_parse/arg_parse_argument.h#L686).

Reply:
All good, I already merged it 😀